resources/lib/kodiutils.py

```diff
@@ -409,6 +410,24 @@ def get_global_setting(key):
     return result.get('result', {}).get('value')
 
 
+def get_property(key, window_id=10000):
+    ''' Get a Window property '''
+    from xbmcgui import Window
+    return Window(window_id).getProperty(key)
```

Comment:
For Up Next I already improved upon this:
```suggestion
    return to_unicode(Window(window_id).getProperty(key))
```